packages/app-builder-backend/src/eject/templates/packageJson.ts

```diff
@@ -0,0 +1,43 @@
+export const generatePackageJson = (name: string) =>
+  JSON.stringify(
+    {
+      name,
+      version: '0.1.0',
+      private: true,
+      dependencies: {
+        react: '^17.0.2',
+        'react-dom': '^17.0.2',
+        'react-scripts': '4.0.3',
+        '@apollo/client': '^3.4.7',
+        'styled-components': '^5.3.1',
+        graphql: '^15.5.1',
+        'react-router': '^5.2.0',
+        'react-router-dom': '^5.2.0',
+        typescript: '^4.4.2',
+
+        'omit-deep': '^0.3.0',
+        'query-string': '^7.0.1',
+        'clone-deep': '^4.0.1',
+
+        '@reapit/connect-session': '^3.0.0', // TODO: should update dynamically to latest version
```

Comment:
Can you just use @latest ? we flag our NPM releases

Reply:
unfortunately when you do that it comes back with this :/ 
![image](https://user-images.githubusercontent.com/1060378/132865968-2e3ef35f-787e-488b-b5ff-08c5389b82f8.png)